config/taxonomy/Makefile

```diff
@@ -0,0 +1,9 @@
+GLANG ?= "go"
+all: clean compile gen
+compile:
+	go run ../../main.go taxonomy compile --base ./base/base.yaml --out ./objects/taxonomy.json \
```

Comment:
Add `ROOT_DIR := ../..` and use it instead of `../..` like in other Makefiles.

Reply:
Done